es-vue-base/src/lib-components/EsNavBarSearchBar.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <div class="align-items-center w-100 d-flex justify-content-center position-relative">
+        <div class="d-flex align-items-center mx-auto justify-content-center w-100">
+            <es-form-input
+                id="searchBar"
+                class="mt-3 w-50"
+                v-bind="$attrs"
+                label-sr-only
+                :placeholder="label"
+                v-on="$listeners">
```

Comment:
Right now clicking "enter" doesn't submit the form, which I imagine is something we want.. We can fix easily by emitting the event when that happens as well